services/site/tests/components/challenge/Editors.spec.tsx

```diff
@@ -25,21 +25,23 @@ const handleReset = jest.fn();
 
 describe("Editors", () => {
   it("renders the wrapper element with classes", () => {
-    const wrapper = shallow(<Editors className={mockClassName} editors={[htmlEditorConfig]} onReset={handleReset} />);
+    const { container } = render(<Editors className={mockClassName} editors={[htmlEditorConfig]} onReset={handleReset} />);
 
-    expect(wrapper.exists("div")).toBeTruthy();
-    expect(wrapper.find("div").first().props().className).toContain(mockClassName);
+    // eslint-disable-next-line testing-library/no-node-access, testing-library/no-container
+    expect(container.querySelector(`div.${mockClassName}`)).toBeInTheDocument();
   });
 
-  it("renders one editor", () => {
-    const wrapper = shallow(<Editors className={mockClassName} editors={[htmlEditorConfig]} onReset={handleReset} />);
+  it("can render a single editor", () => {
```

Comment:
I would maybe keep the wording for tests in this file at "renders XYZ", as all other tests in the project start that way.